lib/demos/material/cards_demo.dart

```diff
@@ -373,13 +368,27 @@ class TravelDestinationContent extends StatelessWidget {
 }
 
 class CardsDemo extends StatefulWidget {
-  const CardsDemo();
+  const CardsDemo({
+    this.restorationId,
+  });
+
+  final String restorationId;
```

Comment:
Does the restorationId need to be a parameter to the constructor? Could it just be a const for each demo instead?

Reply:
If each demo is only ever used once, that would work just fine and could simplify things.